Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/ConceptSearchParameterValidator.java

```diff
@@ -2,10 +2,19 @@
 
 import org.springframework.stereotype.Component;
 import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
 import org.springframework.validation.Validator;
 
 import edu.asu.conceptpower.servlet.core.POS;
 
+/**
+ * This class validates the Concept power search parameters. Checks whether user
+ * has submitted word and pos details for searching the concept In addition, the
+ * validator checks for the operators (OR and AND)
```

Comment:
- Conceptpower is one word. 
- Should be "Checks whether **a** user has submitted word and pos details **when** searching for a concept**.**" (period is missing at the end)
- periods is missing at the end of last sentence as well.